src/main/java/com/hivemq/cli/commands/swarm/commander/SwarmStatusCommand.java

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright 2019 HiveMQ and the HiveMQ Community
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hivemq.cli.commands.swarm.commander;
+
+import com.google.gson.Gson;
+import com.hivemq.cli.MqttCLIMain;
+import com.hivemq.cli.commands.swarm.AbstractSwarmCommand;
+import com.hivemq.cli.commands.swarm.error.Error;
+import com.hivemq.cli.commands.swarm.error.SwarmApiErrorTransformer;
+import com.hivemq.cli.openapi.ApiException;
+import com.hivemq.cli.openapi.swarm.CommanderApi;
+import com.hivemq.cli.openapi.swarm.CommanderStateResponse;
+import com.hivemq.cli.openapi.swarm.RunResponse;
+import com.hivemq.cli.openapi.swarm.RunsApi;
+import okhttp3.HttpUrl;
+import org.jetbrains.annotations.NotNull;
+import org.tinylog.Logger;
+import picocli.CommandLine;
+
+import javax.inject.Inject;
+import javax.inject.Provider;
+import java.io.PrintStream;
+
+/**
+ * @author Yannick Weber
+ */
+@CommandLine.Command(name = "status",
+        description = "Check the status of HiveMQ Swarm.",
+        synopsisHeading = "%n@|bold Usage:|@  ",
+        descriptionHeading = "%n",
+        optionListHeading = "%n@|bold Options:|@%n",
+        commandListHeading = "%n@|bold Commands:|@%n",
+        mixinStandardHelpOptions = true,
+        versionProvider = MqttCLIMain.CLIVersionProvider.class)
+public class SwarmStatusCommand extends AbstractSwarmCommand {
+
+    public enum OutputFormat {
+        JSON, PRETTY
+    }
+
+    @CommandLine.Option(names = {"--format"}, defaultValue = "pretty", description = "The export output format (default pretty)", order = 2)
```

Comment:
The default value may not work because PicoCLI uses case-sensitivity by default.

```suggestion
    @CommandLine.Option(names = {"--format"}, defaultValue = "PRETTY", description = "The export output format (default pretty)", order = 2)
```

Reply:
i disabled case sensitivity.

Reply:
yeah but consistency still is nice